Gemfile

```diff
@@ -72,3 +72,5 @@ group :test do
 end
```

Comment:
![image](https://user-images.githubusercontent.com/93039359/221217280-6cb19a15-6db3-4221-adfd-586a29684ea6.png)

- [ ] It would be nice if you add the following commands in their respective section in your readme file to make your readme file more descriptive and useful for the viewers of your repository :+1:

## Getting Started

### Prerequisites

```
Ruby
Ruby on Rails
Visual Studio Code
PostgreSQL

```
### Setup

```
git clone https://github.com/amexabee/blog-app
cd blog-app
```

## Install

```
bundle install
rails db:create
rails db:migrate

```
## Usage

```
rails server
```
## Test
```
rspec spec
```